CMakeLists.txt

```diff
@@ -205,10 +205,11 @@ if (MATRIX_DOC_PATH AND GTAD_PATH)
     )
     if (ABS_CLANG_FORMAT)
         # TODO: list(TRANSFORM) is available from CMake 3.12
-        foreach (S ${api_SRCS})
-            string (REGEX REPLACE ".cpp$" ".h" H ${S})
-            list(APPEND api_HDRS ${H})
-        endforeach()
+        # foreach (S ${api_SRCS})
+        #    string (REGEX REPLACE ".cpp$" ".h" H ${S})
+        #    list(APPEND api_HDRS ${H})
+        # endforeach()
+        list(TRANSFORM api_SRCS REPLACE ".cpp$" ".h" OUTPUT_VARIABLE api_HDRS)
```

Comment:
Please don't leave the old code commented out. If for whatever reason it has to be returned - Git has all the commits. Just replace the old code with new code.

Reply:
Yes Sir. Thank you.

I dont understand why build is failing. It seems that a particular object file `lib/events/roompowerlevelsevent.o` is not being included in the object files. Why is this happening?

Reply:
That's unrelated to your changes, fortunately - one of previous merges has broken the CI (that includes not only CMake- but also qmake-based pipeline) and it's not fixed yet.

Reply:
Sir, I would like to fix the CI problem, could you please point me to the merges that broke the pipeline. I would like to give this a try. (Mainly, because I want to learn how to do CI/CD)

Reply:
Sir, i amy trying to fix Ci in this PR #378